examples/hosts/iframe-host/src/outer.ts

```diff
@@ -31,40 +30,37 @@ const getTinyliciousUrlResolver =
         "tinylicious",
         "bearer");
 
-export async function loadFrame(iframeId: string, logId: string) {
+export async function loadFrame(iframeDivId: string, divId: string, logId: string) {
     const documentId = getDocumentId();
-    const iframe = document.getElementById(iframeId) as HTMLIFrameElement;
+    const iframeDiv = document.getElementById(iframeDivId) as HTMLIFrameElement;
+    const iframe = document.createElement("iframe");
+    iframe.src = "/inner.html";
+    iframeDiv.appendChild(iframe);
 
     const urlResolver = getTinyliciousUrlResolver();
 
     const tokenProvider = new InsecureTokenProvider("tinylicious", documentId, "12345", { id: "userid0" });
     const documentServiceFactory = new RouterliciousDocumentServiceFactory(tokenProvider);
 
+    const module = { fluidExport: TodoContainer };
+    const codeLoader = { load: async () => module };
```

Comment:
the outer should never need the real code, the whole point is to isolate the real code in the iframe